packages/data-provider/src/repository/submittedRepository.ts

```diff
@@ -16,7 +43,18 @@ const repository = (dependencies: BaseDependencies) => {
 		isValid: true,
 		systemId: true,
 	};
+
 	return {
+		delete: async (data: SubmittedData, comment: string, userName: string) => {
+			const deletedRecord = await db.delete(submittedData).where(eq(submittedData.id, data.id));
+
+			if (audit.enabled) {
```

Comment:
Reading from a env variable. default is true, but if set to false no audit records are stored